jsonschema-hyperschema.xml

```diff
@@ -442,8 +459,17 @@
 
         <section title="Link Description Object">
             <t>
-                A Link Description Object (LDO) is used to describe a single link relation from the
-                instance to another resource.
+                A Link Description Object (LDO) is a serialization of the abstract model specified
+                in Section 2 of <xref target="I-D.nottingham-rfc5988bis"/>, defines a link as
```

Comment:
I'm not sure the comma after the `<xref .../>` is appropriate. As is, I interpret the "defines" verb with "Link Description Object" as subject, is it intended? Or should it `in Section 2 of <xref target="I-D.nottingham-rfc5988bis"/> which defines a link as [...]`?